vxyowsup/tests/test_whatsapp.py

```diff
@@ -17,20 +18,26 @@
     TextMessageProtocolEntity)
 
 
-class DummyStackBuilder(YowStackBuilder):
-    @staticmethod
-    def getCoreLayers():
-        return (
-            YowLoggerLayer,
-            TestingLayer,
-        )[::-1]
+@staticmethod
+def getDummyCoreLayers():
+    return (
+        YowLoggerLayer,
+        TestingLayer,
+    )[::-1]
```

Comment:
I wonder whether we should just drop the logging layer for now and perhaps also remove the somewhat cryptic `[::-1]` reversing of the tuple?